pkgs/development/libraries/art-standalone/default.nix

```diff
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  wolfssl,
+  bionic-translation,
+  python3,
+  which,
+  jdk17,
+  zip,
+  xz,
+  icu,
+  zlib,
+  libcap,
+  expat,
+  openssl,
+  libbsd,
+  lz4,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "art-standalone";
+  version = "0-unstable-2024-12-01";
+
+  src = fetchFromGitLab {
+    owner = "android_translation_layer";
+    repo = "art_standalone";
+    rev = "c4fba44a45e319b57801d81ef946ee953ff70b58";
+    hash = "sha256-bNtSNmWRDGsl1/AUC0t0xII1DCH3uKJwhtz0b2nW9qw=";
+  };
+
+  patches = [ ./add-liblog-dep.patch ];
+
+  enableParallelBuilding = true;
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    jdk17
+    python3
+    which
+    zip
+  ];
+
+  buildInputs = [
+    bionic-translation
+    expat
+    icu
+    libbsd
+    libcap
+    lz4
+    openssl
+    wolfssl
```

Comment:
AFAIR not using a specifically JNI-enabled wolfssl (built with `-enable-jni`) breaks something down the line